File: Selenium/Activities/Activity6.py
from selenium import webdriver
from selenium.webdriver import ActionChains, Keys
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

serv = FirefoxService(GeckoDriverManager().install())

with webdriver.Firefox(service=serv) as driver:
    driver.get("https://training-support.net/selenium/input-events")
    print("Title is: ", driver.title)
    action = ActionChains(driver)
    action.send_keys("N")
    action.key_down(Keys.CONTROL)
    action.send_keys("a")
    action.send_keys("c")
    action.key_up(Keys.CONTROL)
    action.perform()
    driver.quit()

File: Selenium/Activities/Activity6.java
package Activities;
import org.openqa.selenium.By;
import org.openqa.selenium.Keys;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.openqa.selenium.interactions.Actions;


public class Activity6 {
    public static void main(String[] args)
    {
        WebDriverManager.firefoxdriver().setup();
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE,"/dev/null");
        WebDriver newDriver = new FirefoxDriver();
        newDriver.get("https://training-support.net/selenium/input-events");
        System.out.println("Title is: "+newDriver.getTitle());
        Actions action = new Actions(newDriver);
        action.sendKeys("N").pause(1000).build().perform();
        action.keyDown(Keys.CONTROL).sendKeys("a").sendKeys("c").keyUp(Keys.CONTROL).pause(1000).build().perform();
        newDriver.quit();
    }
}
